builtin/common/vector.lua

```diff
@@ -4,12 +4,13 @@ vector = {}
 function vector.new(a, b, c)
 	if type(a) == "table" then
 		assert(a.x and a.y and a.z, "Invalid vector passed to vector.new()")
-		return {x=a.x, y=a.y, z=a.z}
+		return {x = a.x, y = a.y, z = a.z}
+	elseif a and b and c then
```

Comment:
There's a problem here if a mod has a bug, and it calls:

```
vector.new(1, null, 3)
```

so this isn't a good change